wfe/wfe.go

```diff
@@ -1669,3 +1664,10 @@ func (wfe *WebFrontEndImpl) addIssuingCertificateURLs(response http.ResponseWrit
 	}
 	return nil
 }
+
+func urlForAuthz(authz core.Authorization, request *http.Request) string {
```

Comment:
Rather than duplicating this function in the WFE and the WFE2 can you put it into `web` alongside `RelativeEndpoint` as an exported function? It looks like the `authzPath`, and `authz2Prefix` constants are the same in both `wfe` and `wfe2`.

Reply:
Thinking about it more it might just be a mess. I switched to +1. These functions are super small and we'll :hocho: the wfe1 entirely sooner or later...